hs_geographic_feature_resource/receivers.py

```diff
@@ -212,51 +212,65 @@ def geofeature_pre_create_resource(sender, **kwargs):
     tmp_dir = None
     try:
         files = kwargs['files']
-        metadata = kwargs['metadata']
-        validate_files_dict = kwargs['validate_files']
         fed_res_fnames = kwargs['fed_res_file_names']
         fed_res_path = kwargs['fed_res_path']
-        file_selected = False
-
-        if files:
-            file_selected = True
-            file_list = [file.name for file in files]
-        elif fed_res_fnames:
-            file_list = utils.get_fed_zone_files(fed_res_fnames)
-            if file_list:
-                file_selected = True
-
-        if file_selected:
-            files_type = check_uploaded_files_type(files, file_list)
+        metadata = kwargs['metadata']
+        validate_files_dict = kwargs['validate_files']
+
+        if files and fed_res_fnames:
+            validate_files_dict['are_files_valid'] = False
+            validate_files_dict['message'] = 'Please upload files from either local disk or irods, not both.'
+            return
+
+        if files or fed_res_fnames:
+            files_list = [f for f in files]
```

Comment:
@pkdash After chatting with @zhiyuli, @zhiyuli decided to further work on removing the unneeded step of wrapping the file retrieved from a federation zone into UploadedFile() object, so I will not add another function as proposed above. We plan to integrate work together by midday tomorrow at the latest, so @mjstealey you and me can do a pre-deploy release 1.7.2 on beta tomorrow afternoon in time.